### PR TITLE
Remove characters class

### PR DESCRIPTION
**Pull request by cphyc**
Removed characters class and recreated as Y.List.
Tests are still required.

### CODE REVIEW

File: lib/characters.coffee [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Y = require '../../yjs/lib/y.coffee'
+Y.List = require '../../y-list/lib/y-list.coffee'
+
+
 class Characters
```
Comment (dmonad) [inline lib/characters.coffee]: still here?

File: lib/misc.coffee [hunk 2]
```diff
@@ -21,11 +21,8 @@ class BaseClass
       @_model.val(prop, val)
 
   # since we already assume that any instance of BaseClass uses a MapManager
-  # We can create it here, to save
-  # lines
-  # of
-  # code
-  _setModel: (@_model)->
+  # We can create it here, to save lines of code
+  _getModel: (@_model)->
```
Comment (dmonad) [inline lib/misc.coffee]: setModel!

File: lib/characters.coffee (hunk 1 above)
Comment (cphyc) [inline lib/characters.coffee]: Nope, I just removed it.

File: lib/y-richtext.coffee [hunk 3]
```diff
@@ -1,38 +1,49 @@
...
+      @_setCursor @editor.getCursorPosition()
+      @_setModel @_model
 
       # listen to events on the model using the function propagateToEditor
       @_model.observe @propagateToEditor
     return @_model
 
   _setModel: (model) ->
-    super
+    super model
```
Comment (dmonad) [inline lib/y-richtext.coffee]: just super here! This is much easier to maintain!

```
super
```

File: lib/misc.coffee [hunk 4]
```diff
@@ -21,15 +21,12 @@ class BaseClass
...
-  # code
-  _setModel: (@_model)->
+  # We can create it here, to save lines of code
+  _getModel: (Y, Operation)->
     if not @_model?
       @_model = new Operation.MapManager(@).execute()
-      for n,v of @_tmp_model
-        @_model.val(n,v)
+      for key, value of @_tmp_model
+        @_model.val(key, value)
```
Comment (cphyc) [inline lib/misc.coffee:29]: Better?

**Pull request merged**